spotlight-client/src/demographics.ts

```diff
@@ -0,0 +1,81 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { DemographicFieldKey, DemographicView, TOTAL_KEY } from "./metricsApi";
+
+export const DEMOGRAPHIC_UNKNOWN = "EXTERNAL_UNKNOWN";
```

Comment:
Do you ever see any `INTERNAL_UNKNOWN` values?

Reply:
rarely ... I think they are generally the result of errors somewhere in the data pipeline (or bad source data)? We weren't always handling them consistently in the original Spotlight so it got a little messy sometimes. (We may want to revisit the approach but I don't think it's an MVP priority considering how rare it is.)

Basically the goal was that we could handle the value and display it properly if it showed up (mainly in the top section of each page that shows all the demographic breakdowns at once), but when we were building layouts around a specific demographic lens (e.g. a chart for each racial/ethnic category), we didn't want to treat it as one of those categories. The latter part is what's happening here.